contribution.md

```diff
@@ -30,7 +30,7 @@ The original Lua bindings are too complicated at the moment. The idea is to unif
 
 ### Additional statistics backends
 
-Currently, we support plain mmapped files only for storing statistics tokens. However, we'd like to support other backends, such as redis, sqlite or other storages.
```

Comment:
Probably it should be spelled `memory-mapped`.

It is of course ancient history.

Reply:
Thanks, this one the one change I was not sure about